sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -115,6 +115,30 @@ void gnrc_ipv6_nib_init(void)
 
 static void _add_static_lladdr(gnrc_netif_t *netif)
 {
+#if defined(CONFIG_GNRC_IPV6_STATIC_LLADDR_NETDEV_MASK) && \
+    CONFIG_GNRC_IPV6_STATIC_LLADDR_NETDEV_MASK > 0
+#ifndef MODULE_NETDEV_REGISTER
+#error "Use of CONFIG_GNRC_IPV6_STATIC_LLADDR_NETDEV_MASK requires MODULE_NETDEV_REGISTER"
+#endif
+    if (! ((CONFIG_GNRC_IPV6_STATIC_LLADDR_NETDEV_MASK & (1ULL << NETDEV_ANY)) ||
+          (CONFIG_GNRC_IPV6_STATIC_LLADDR_NETDEV_MASK & (1ULL << netif->dev->type)))) {
+        DEBUG("nib: interface #%u: not setting static link-local address "
+                "(netdev type %u not included)\n",
+                netif->pid, netif->dev->type);
+        return;
+    }
+#endif
+#if defined(CONFIG_GNRC_IPV6_STATIC_LLADDR_IFACE_MASK) && \
+    CONFIG_GNRC_IPV6_STATIC_LLADDR_IFACE_MASK > 0
+    if (! (CONFIG_GNRC_IPV6_STATIC_LLADDR_IFACE_MASK & (1ULL << netif->pid))) {
```

Comment:
```suggestion
    if (CONFIG_GNRC_IPV6_STATIC_LLADDR_IFACE_MASK && !(CONFIG_GNRC_IPV6_STATIC_LLADDR_IFACE_MASK & (1ULL << netif->pid))) {
```

You can drop the preprocessor conditional

Reply:
this option to select by interface was removed